cirq/aqt/aqt_sampler.py

```diff
@@ -29,19 +29,19 @@
 
 import numpy as np
 from requests import put
-from cirq import circuits, Sampler, resolve_parameters, LineQubit
-from cirq.study.sweeps import Sweep
-from cirq.aqt.aqt_device import AQTSimulator, get_op_string
-from cirq import study, ops, IonDevice
+
+import cirq
```

Comment:
@Strilanc what's the convention for submodules importing cirq directly? I think you're still "supposed" to 

```python
from cirq import circuits, ops, protocols
```

etc.

Reply:
ping @Strilanc

Reply:
xref https://github.com/quantumlib/Cirq/issues/2787 

Unless someone tells me otherwise, I don't think you're supposed to `import cirq` at the top level for any "internal" code (that's part of cirq)

Reply:
We should not "import cirq" from within non-test python files within cirq.

In tests, you can "import cirq"